tests/h/search/query_test.py

```diff
@@ -3,188 +3,182 @@
-        # Search appends by default.
-        return search.Search(pyramid_request)
+    def filtered_search(self, search):
+        search.append_filter(query.UserFilter())
+        return search
 
 
 class TestUriFilter(object):
 
     @pytest.fixture
-    def search(self, pyramid_request):
-        # We don't need to append UriFilter to Search because it's one of the filters that
-        # Search appends by default.
-        return search.Search(pyramid_request)
+    def filtered_search(self, search, pyramid_request):
+        search.append_filter(query.UriFilter(pyramid_request))
+        return search
 
 
 class TestDeletedFilter(object):
 
     @pytest.fixture
-    def search(self, pyramid_request):
-        # We don't need to append DeletedFilter to Search because it's one of the filters that
-        # Search appends by default.
-        return search.Search(pyramid_request)
+    def filtered_search(self, search):
+        search.append_filter(query.DeletedFilter())
+        return search
 
 
 class TestNipsaFilter(object):
 
     @pytest.fixture
-    def search(self, pyramid_request):
-        # We don't need to append NipsaFilter to Search because it's one of the filters that
-        # Search appends by default.
-        return search.Search(pyramid_request)
+    def filtered_search(self, search, pyramid_request):
+        search.append_filter(query.NipsaFilter(pyramid_request))
+        return search
 
 
 class TestAnyMatcher(object):
 
     @pytest.fixture
-    def search(self, pyramid_request):
-        # We don't need to append AnyMatcher to Search because it's one of the matchers that
-        # Search appends by default.
-        return search.Search(pyramid_request)
+    def filtered_search(self, search):
+        search.append_matcher(query.AnyMatcher())
+        return search
 
 
 class TestTagsMatcher(object):
 
     @pytest.fixture
-    def search(self, pyramid_request):
-        # We don't need to append TagsMatcher to Search because it's one of the matchers that
-        # Search appends by default.
-        return search.Search(pyramid_request)
+    def filtered_search(self, search):
+        search.append_matcher(query.TagsMatcher())
+        return search
 
 
 class TestRepliesMatcher(object):
 
     # Note: tests will have to append a RepliesMatcher object to the search
     # (search.append_matcher(RepliesMatcher(annotation_ids))) passing to RepliesMatcher the
     # annotation_ids of the annotations that the test wants to search for replies to.
-
-    @pytest.fixture
-    def search(self, pyramid_request):
-        return search.Search(pyramid_request)
+    pass
 
 
 class TestTagsAggregation(object):
 
     @pytest.fixture
-    def search(self, pyramid_request):
-        search_ = search.Search(pyramid_request)
-        search_.append_aggregation(search.TagsAggregation())
-        return search_
+    def filtered_search(self, search):
+        search.append_filter(query.TagsAggregation())
```

Comment:
This should be `append_aggregation` surely? Same with `TestUsersAggregation` below.